upgrade/v120to121/upgrade.go

```diff
@@ -0,0 +1,77 @@
+package v120to121
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/longhorn/longhorn-manager/types"
+
+	lhclientset "github.com/longhorn/longhorn-manager/k8s/pkg/client/clientset/versioned"
+)
+
+const (
+	upgradeLogPrefix = "upgrade from v1.2.0 to v1.2.1: "
+)
+
+func UpgradeCRs(namespace string, lhClient *lhclientset.Clientset) (err error) {
+	defer func() {
+		err = errors.Wrapf(err, upgradeLogPrefix+"UpgradeCRs failed")
+	}()
+	if err := upgradeSettings(namespace, lhClient); err != nil {
+		return err
+	}
+	return nil
+}
+
+func upgradeSettings(namespace string, lhClient *lhclientset.Clientset) (err error) {
+	defer func() {
+		err = errors.Wrapf(err, "upgrade settings failed")
+	}()
+
+	// Skip the upgrade if this deprecated setting is unavailable.
+	disableReplicaRebuildSetting, err := lhClient.LonghornV1beta1().Settings(namespace).Get(context.TODO(), string(types.SettingNameDisableReplicaRebuild), metav1.GetOptions{})
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			return nil
+		}
+		return err
+	}
+	disabled, err := strconv.ParseBool(disableReplicaRebuildSetting.Value)
+	if err != nil {
+		logrus.Warnf(upgradeLogPrefix+"Failed to parse boolean value for setting %v, will skip checking it, error: %v", types.SettingNameDisableReplicaRebuild, err)
+		return nil
+	}
+	if disabled {
```

Comment:
This should be `if !disabled`. You need to skip the following concurrent setting update if the disabled is false.

Reply:
Yes. If this deprecated setting zs not modified, the concurrent setting should be the default value or the customized value.